.buildkite/smart_exhaustive_tests_pipeline.yml

```diff
@@ -0,0 +1,33 @@
+steps:
+  - label: "Trigger logstash-exhaustive-tests-pipeline for PRs with qa/acceptance/ changes"
+    if: build.pull_request.id != null
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          interpolation: false
+          watch:
+            - path:
+                - ^qa/acceptance/
```

Comment:
IMO just having the command to trigger these tests (instead of having to go to buildkite to kick off a run for a PR) will cover the cases we care about. However, i think having a trigger on certain files is a decent pattern to introduce if for no other reason than to establish the precedent. 

We could also look at this file too https://github.com/elastic/logstash/blob/main/ci/acceptance_tests.sh

Reply:
No strong opinion here, if we only use GitHub commands, then we can reduce the complexity with the new pipeline